### DISCUSSION TITLE
slider step size 5  Burner power changes  v2.8.3-beta

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1165] [General]
Keyboard operation slider step 5 burner power change, there is a problem.

https://github.com/artisan-roaster-scope/artisan/assets/116416332/14043cf9-7978-4748-a576-e8fc92844055

**Comment by Terracotta-62** [comment]
[artisan-settings.zip](https://github.com/artisan-roaster-scope/artisan/files/11456685/artisan-settings.zip)

**Comment by MAKOMO** [comment]
Wow! Very good find! Thanks for reporting this very first 2.8.3-beta bug! Had a typo in setting up Slider 4, which should be fixed now in the Continuous build (a70bfe34bd4bf932dcf0b8edb0a9245a7ca15df9).
